docs/intro.rst

```diff
@@ -48,7 +48,14 @@ following commands in your command prompt/terminal emulator.
       sudo apt-get build-dep python-lxml
       sudo apt-get install libffi-dev
       sudo apt-get install libssl-dev
-
+ 
```

Comment:
Just leaving a note for myself that this line begins with a space (and probably shouldn't but oh well), hence the +/- of a newline here.